types/Anchor.d.ts

```diff
@@ -0,0 +1,5 @@
+type Anchor = {
```

Comment:
I created a `types/` folder so we can share types across the app without having to import anything (which ESLint complains about since it thinks the variable is unused).

For example, we can use this in [#12].